.changeset/wise-hats-smell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/cloudflare': patch
+---
+
+Fixes an issue where internal fetch was not allowed
```

Comment:
```suggestion
Fixes an issue where internal fetch was not allowed, preventing setting values for `Astro.response`
```
I'm just wondering whether a user thinks of it as "internal fetch" vs dealing with setting response objects/headers etc. (from reading the issue this closes - they seemed surprised Astro was performing a fetch at all?)

I'm not sure if the above is the correct wording, but I think the user-facing issue is more about their having trouble with `Astro.response`, so this could even be:

> Fixes an issue (something about `Astro.response`, or setting 404 status...)